src/components/calendar/common/calendar-base.ts

```diff
@@ -105,4 +105,12 @@ export class IgcCalendarBaseComponent extends LitElement {
   protected selectionChange() {
     this.value = undefined;
   }
+
+  protected override async firstUpdated() {
+    if (this.selection === 'single') {
+      this.activeDate = this.value ? this.value : new Date();
```

Comment:
you can simplify this with [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)